tests/Api/Utils/ShopUserLoginTrait.php

```diff
@@ -13,19 +13,22 @@
 
 namespace Sylius\Tests\Api\Utils;
 
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+
 trait ShopUserLoginTrait
 {
-    protected function logInShopUser(string $email, string $password): array
+    protected function logInShopUser(string $email): string
     {
-        $this->client->request(
```

Comment:
As it has been removed, it would be nice to add a contract test for signing in as a shop user to check customer iri